test/extensions/filters/http/cors/cors_filter_integration_test.cc

```diff
@@ -66,6 +66,15 @@ class CorsFilterIntegrationTest : public HttpIntegrationTest,
             auto* cors = route->mutable_route()->mutable_cors();
             cors->add_allow_origin_regex(".*\\.envoyproxy\\.io");
           }
+
+          {
+            auto* route = virtual_host->add_routes();
+            route->mutable_match()->set_prefix("/cors-expose-headers");
+            route->mutable_route()->set_cluster("cluster_0");
+            auto* cors = route->mutable_route()->mutable_cors();
+            cors->add_allow_origin("test-origin-1");
+            cors->set_expose_headers("custom-header-1");
```

Comment:
Nit, can we have a comma-separated list of headers here (e.g. `X-My-Custom-Header, X-Another-Custom-Header`)? Just to make a point that we can express setting multiple headers by doing so.

Reply:
@dio, thanks for the review. I have updated the test with multiple expose-headers.

Should I also take out the setting of `access-control-expose-headers` in the `OPTIONS` request in this same PR?